cns/service/main.go

```diff
@@ -526,7 +526,7 @@ func main() {
 	// Initialze state in if CNS is running in CRD mode
 	// State must be initialized before we start HTTPRestService
 	if config.ChannelMode == cns.CRD {
-		err = InitializeCRDState(httpRestService, cnsconfig)
+		err = InitializeCRDState(rootCtx, httpRestService, cnsconfig)
```

Comment:
If, as per PR title, `rootCtx` is only used in main, does it still need to be a package level variable?

Reply:
it does, because it is created in the `init()`, before `main()` executes. 
this change is groundwork for a refactor of this `main.go` to remove all the helper functions to more appropriate locations in their target packages, leaving only init/main and some minimal global state (rootCtx/cancel) in this file/in the main package.